Makefile

```diff
@@ -548,6 +548,7 @@ secrets/setup/empty:
 	touch $(SECRETS_DIR)/kafka-tls.crt
 	touch $(SECRETS_DIR)/kafka-tls.key
 	touch $(SECRETS_DIR)/image-pull.dockerconfigjson
+	touch $(SECRETS_DIR)/dataplane-observability-config.yaml
```

Comment:
Should the:

	touch $(SECRETS_DIR)/rhsso-logs.clientId
	touch $(SECRETS_DIR)/rhsso-logs.clientSecret
	touch $(SECRETS_DIR)/rhsso-metrics.clientId
	touch $(SECRETS_DIR)/rhsso-metrics.clientSecret

be removed?